R/heatmaply.R

```diff
@@ -571,6 +571,7 @@ heatmaply.default <- function(x,
                               label_format_fun = function(...) format(..., digits = 4),
                               labRow = NULL, labCol = NULL,
                               custom_hovertext = NULL,
+                              suppress_default_hovertext = F,
```

Comment:
Please use FALSE instead of F
```suggestion
                              suppress_default_hovertext = FALSE,
```